CTestConfig.cmake

```diff
@@ -0,0 +1,14 @@
+## This file should be placed in the root directory of your project.
```

Comment:
This file should not be checked-in.   Are you building from the qmcpack/ directory?  You should be building in the build/ directory to keep the directory tree cleaner.

Reply:
This is used by the ci tool I'm using locally to store settings about the project, I'll see about moving it somewhere else.  I'll remove it from here.